### PR TITLE
Release 0.19.0

### PR DESCRIPTION
**Pull request by MALPI**
Set proper release version

### CODE REVIEW

**Comment by MALPI** [comment]
:+1:

**Comment by otrosien** [comment]
:+1:

File: fahrschein-example/pom.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <parent>
         <groupId>org.zalando</groupId>
         <artifactId>fahrschein-parent</artifactId>
-        <version>0.20.0-SNAPSHOT</version>
+        <version>0.19.0</version>
```
Comment (othon2000) [inline fahrschein-example/pom.xml:10]: I think we can remove the `<version>` tag from all submodules, and they will inherit from the parent.

Comment (MALPI) [reply fahrschein-example/pom.xml:10]: Lets consider an extra issue for that.

**Pull request merged**